WalletWasabi.Tests/NodeBuilding/NodeBuilder.cs

```diff
@@ -210,7 +196,7 @@ public void Dispose()
 		{
 			foreach (CoreNode node in Nodes)
 			{
-				node.Kill();
+				node.KillAsync().GetAwaiter().GetResult();
```

Comment:
Why block here?

Reply:
Because I didn't clean the code after my latest change. it is cleaned now.